### ISSUE TITLE
Cannot build PSF for R062 filter in WebbPSF 0.8.0

### ISSUE THREAD

**Issue by Onoddil** [spacetelescope/webbpsf#285]
Despite the documentation saying WebbPSF adds the R062 filter, the aberrations do not appear to support the wavelength range of the filter.  Using the code

```
from webbpsf import wfirst
wfi = wfirst.WFI()
wfi.filter = 'r062'
wfi.detector = 'SCA09'
wfi.detector_position = (2048, 2048)
wfi.options['parity'] = 'odd'
wfi.options['output_mode'] = 'both'

psf = wfi.calc_psf(oversample=4)
```

I first get warning ``The supplied file, $WEBBPSF_PATH/WFI/filters/R062_throughput.fits, does not have a WAVEUNIT keyword. Assuming it is Angstroms.`` and then the error ``RuntimeError: Attempted to get aberrations at wavelength outside the range of the reference data``. This runs correctly if ``wfi.filter = 'z087`` is used instead, which has ``WAVEUNIT= 'angstrom'`` in its header. I believe that the defaulting units are correct, as given by

```
>>> import astropy.io.fits as pyfits
>>> a = pyfits.open('$WEBBPSF_PATH/WFI/filters/R062_throughput.fits')[1].data
>>> a
FITS_rec([( 4000., 0.), ( 4799., 0.), ( 4800., 1.), ( 8000., 1.),
          ( 8001., 0.), (30000., 0.)],
         dtype=(numpy.record, [('WAVELENGTH', '>f8'), ('THROUGHPUT', '>f8')]))
>>> b = pyfits.open('$WEBBPSF_PATH/WFI/filters/Z087_throughput.fits')[1].data
>>> b
FITS_rec([(  5000., 0.), (  7599., 0.), (  7600., 1.), (  9770., 1.),
          (  9771., 0.), (300000., 0.)],
         dtype=(numpy.record, [('WAVELENGTH', '>f4'), ('THROUGHPUT', '>f8')])
```

I have confirmed my module version and webbpsf-data version via ``conda list webbpsf`` giving ``webbpsf 0.8.0 <pip>`` and ``version.txt`` in webbpsf-data also showing ``0.8.0``. Are there any known issues with using the R062 filter in WebbPSF, or any other reasons my PSF creation may have failed? One potential explanation I have found is that the documentation says that the Zernike aberrations only extend to 0.76 microns, but also says that the wavelength range should be extended in Cycle 7. Are my Zernike aberrations incorrect in some way; is there a more up to date table of these values that I do not have in my webbpsf-data?

**Comment by mperrin** [comment]
@robelgeda didn’t you fix this already? Can you remind me the details - does this need just the updated data files?

**Comment by mperrin** [comment]
@Onoddil we believe this has been fixed in the latest code, so you will want to (a) install from master beret n GitHub rather than the release and (b) get an updated data file from either Robel or I (available right after lunch once I’m at a real computer rather than a phone).

**Comment by mperrin** [comment]
@Onoddil stand by, prior comment was incorrect and this does actually appear to need a quick fix.

**Comment by mperrin** [comment]
@Onoddil there's a fix for this in a PR now that you could try; should be tested and merged into master by later today I hope.

**Comment by mperrin** [comment]
@rebekah9969 tells me that for Cycle 6, she and Joseph Long wrote code to extrapolate the Zernike dependence from longer wavelengths down to shorter wavelengths. This is more precise than just a flat extrapolation of constant WFE like we were doing.  She is going to try to find the code that they did this with before.

**Comment by mperrin** [comment]
Writing down here some of the history that I reconstructed today: 
  - for the Cycle 6 data delivery, Rebekah and Joseph developed an extrapolation in Zernike space of the wavelength dependence from longer to shorter wavelengths. That was used to make a broader zernike lookup table file
 - When that lookup table was replaced with the Cycle 7 values, that same process wasn't applied (because totally different people were involved, and the data delivery from Goddard wasn't even consistently formatted with the prior one)

Plan forward: 
1. We'll merge some short term workaround so that the code will run without erring and give reasonable results, by just treating WFE as constant outside of the modeled range (timescale ~ next couple days)
2. Rebekah will try to find the earlier notebook they did that extrapolation in; failing that we will just code up the same idea again. That will let us make a better Zernikes lookup table for Cycle 7. (timescale  within a month or so?)
3. Maxime (@mjrfringes) offered to help coordinate a more complete future data release, which should contain rigorous optical model results for the entire WFI wavelength range (timescale ~ whenever cycle 8 modeling is?)
